src/protocols/rdp/channels/audio-input.h

```diff
@@ -296,18 +292,13 @@ guac_user_end_handler guac_rdp_audio_end_handler;
 /**
  * Adds Guacamole's "guacai" plugin to the list of dynamic virtual channel
  * plugins to be loaded by FreeRDP's "drdynvc" plugin. The plugin will only
- * be loaded once guac_rdp_load_drdynvc() is invoked with the guac_rdp_dvc_list
- * passed to this function. The "guacai" plugin ultimately adds support for the
- * "AUDIO_INPUT"  dynamic virtual channel.
+ * be loaded once the "drdynvc" plugin is loaded. The "guacai" plugin
+ * ultimately adds support for the "AUDIO_INPUT"  dynamic virtual channel.
```

Comment:
Double space in this comment should be removed.

Reply:
Should now be fixed.